versioned_plugins.rb

```diff
@@ -429,6 +432,41 @@ def load_alias_definitions_for_target_plugins(plugin_names_by_type)
 
     aliases
   end
+
+  def fetch_stack_versions
+    stack_versions = Net::HTTP.get(URI.parse(VERSIONS_URL))
+    puts "Logstash version: #{get_logstash_version(stack_versions)}\n"
+    puts "ECS version: #{get_ecs_version(stack_versions)}\n"
+  end
+
+  def get_logstash_version(stack_versions)
+    version = stack_versions[/\:logstash_version: (.*?)\n/, 1]
+    @logstash_version = version.strip unless version.nil?
```

Comment:
If we tell the regex that there will be multiple whitespaces instead of just one we don't need to strip it:

```suggestion
    @logstash_version = stack_versions[/\:logstash_version:\s+(.*?)\n/, 1]
```